flixel/system/scaleModes/shaders/ScaleShaderFilter.hx

```diff
@@ -0,0 +1,42 @@
+package flixel.system.scaleModes.shaders;
+
+import openfl.display.Shader;
+import openfl.filters.BitmapFilter;
+import openfl.geom.Matrix;
+import openfl.geom.Rectangle;
+import openfl._internal.renderer.RenderSession;
+
+
+class ScaleShaderFilter extends openfl.filters.ShaderFilter {
+	
+	public var scaleX:Float;
+	public var scaleY:Float;
+	public var resolution:Array<Float>;
+
+	public function new(shader:Shader) {
+		super(shader);
+	}
+	
+	override public function clone():BitmapFilter {
+		return super.clone();
+	}
+	
+	override function __growBounds (rect:Rectangle) {
```

Comment:
This was the original function in the super class. As far as I know it is just called once before the shader is applied unless the `__passes` variable is increased. It maintains functionality.
```haxe
override function __preparePass(pass:Int):Shader {
		return shader;
}
```
Perhaps it should return `super.__preparePass(pass);`?